applovin_max/android/src/main/java/com/applovin/applovin_max/AppLovinMAX.java

```diff
@@ -957,28 +957,8 @@ else if ( mAppOpenAds.containsKey( adUnitId ) )
             return;
         }
 
-        fireErrorCallback( name, adUnitId, error, sharedChannel );
-    }
-
-    public void fireErrorCallback(final String name, final String adUnitId, final MaxError error, final MethodChannel channel)
-    {
-        try
-        {
-            Map<String, Object> params = new HashMap<>( 5 );
-            params.put( "adUnitId", adUnitId );
-
-            if ( error != null )
-            {
-                params.put( "errorCode", error.getCode() );
-                params.put( "errorMessage", error.getMessage() );
-                params.put( "waterfall", createAdWaterfallInfo( error.getWaterfall() ) );
-            }
-            else
-            {
-                params.put( "errorCode", MaxErrorCode.UNSPECIFIED );
-            }
-
-            fireCallback( name, params, channel );
+        try {
+            fireCallback( name, getAdLoadFailedInfo( adUnitId, error ) );
         }
         catch ( Throwable ignored ) { }
```

Comment:
I would remove the `try-catch`. If you insist on keeping it, I'd put it in `fireCallback(...)`, but if there are exceptions being thrown it'd be nice to know what they are and why they happen so I'm leaning towards just removing.